backend/apps/slack/events/event.py

```diff
@@ -35,25 +33,113 @@ def configure_events():
             logger.warning("SlackConfig.app is None. Event handlers are not registered.")
             return
 
-        for event_class in EventBase.get_events():
-            event_class().register()
+        for event in EventBase.get_events():
+            event().register()
 
     @staticmethod
     def get_events():
         """Yield all subclasses of EventBase."""
         yield from EventBase.__subclasses__()
 
     @property
-    def ephemeral_message(self) -> tuple | None:
-        """Return ephemeral message text."""
+    def direct_message_template(self) -> Template | None:
+        """Get the Jinja template file for direct message.
+
+        Returns:
+            Template | None: The Jinja template object or None.
+
+        """
+        return (
+            env.get_template(str(self.direct_message_template_path))
+            if self.direct_message_template_path
+            else None
+        )
+
+    @property
+    def direct_message_template_path(self) -> Path | None:
+        """Get the Jinja template file path for direct message.
+
+        Returns:
+            Path | None: The Jinja template path or None.
+
+        """
+        return Path(f"events/{convert_to_snake_case(self.__class__.__name__)}.jinja")
+
+    @property
+    def ephemeral_message_template(self) -> Template | None:
+        """Get the Jinja template file for ephemeral message.
+
+        Returns:
+            Template | None: The Jinja template object or None.
+
+        """
+        return (
+            env.get_template(str(self.ephemeral_message_template_path))
+            if self.ephemeral_message_template_path
+            else None
+        )
+
+    @property
+    def ephemeral_message_template_path(self) -> Path | None:
```

Comment:
This always returns `None`. Is that expected behavior? 🤔

Reply:
Added a comment.